golang/explode.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
```suggestion
package pehape
```

Kita bikin library, bukan bikin program

Reply:
pfftttt kirain cuma convert doang. trnyata dibuat package. ok ma bad